frontend/src/components/form/base/ETextField.vue

```diff
@@ -16,10 +16,10 @@
       :required="required"
       :hide-details="hideDetails"
       :error-messages="veeErrors.concat(errorMessages)"
-      :label="label || name"
+      :label="label ?? name"
```

Comment:
Not really related to this PR. But if you touch this, please do it consistently for all form fields.

Reply:
#3892